diff --git a/dist/client/theme-default/components/VPSidebarItem.vue b/dist/client/theme-default/components/VPSidebarItem.vue
index 31ce2dd9907b9911c980220ff19e145b07bd0bc6..c8b47239d1e7c084e69b3ee3021bb25f893b3879 100644
--- a/dist/client/theme-default/components/VPSidebarItem.vue
+++ b/dist/client/theme-default/components/VPSidebarItem.vue
@@ -110,10 +110,6 @@ function onCaretClick() {
 
 <style scoped>
 .VPSidebarItem.level-0 {
-  padding-bottom: 24px;
-}
-
-.VPSidebarItem.collapsed.level-0 {
   padding-bottom: 10px;
 }
 
