diff --git a/dist/client/theme-default/components/VPSidebarItem.vue b/dist/client/theme-default/components/VPSidebarItem.vue
index 59782e0d487c4b77a4c21801839dfa4bdd01eb34..9e3a85ad207587f92a8d34139f7244134b12efcb 100644
--- a/dist/client/theme-default/components/VPSidebarItem.vue
+++ b/dist/client/theme-default/components/VPSidebarItem.vue
@@ -109,10 +109,6 @@ function onCaretClick() {
 
 <style scoped>
 .VPSidebarItem.level-0 {
-  padding-bottom: 24px;
-}
-
-.VPSidebarItem.collapsed.level-0 {
   padding-bottom: 10px;
 }
 
